core-bundle/src/Resources/views/FrontendPreview/authenticator.html.twig

```diff
@@ -0,0 +1,154 @@
+                           name="user"
+                           id="ctrl_user"
+                           list="userlist"
+                           class="tl_text user"
+                           placeholder="{{ 'MSC.username'|trans([], 'contao_default') }}"
+                           value="{{ user }}"
+                           autocomplete="off">
+                    <datalist id="userlist">
+                        <option value="-"></option>
+                    </datalist>
+                {% endif %}
+            </div>
+            <div>
+                <label for="ctrl_unpublished">{{ 'MSC.hiddenElements'|trans([], 'contao_default') }}:</label>
+                <select name="unpublished" id="ctrl_unpublished" class="tl_select">
+                    <option value="hide">{{ 'MSC.hiddenHide'|trans([], 'contao_default') }}</option>
+                    <option value="show"{% if show %} selected{% endif %}>{{ 'MSC.hiddenShow'|trans([], 'contao_default') }}</option>
+                </select>
+                <button type="submit" class="tl_submit">{{ 'MSC.apply'|trans([], 'contao_default') }}</button>
+            </div>
+        </div>
+    </form>
+{% endblock %}
+
+{% block javascript %}
+window.addEventListener("cto-toolbar-loaded", function () {
+    const form = toolbar.querySelector('input[name="FORM_SUBMIT"][value="tl_switch"]').form;
+    if (!form) {
+        return;
+    }
+
+    form.addEventListener('submit', function (e) {
+        e.preventDefault();
+        const formData = new FormData(form);
+
+        request('POST', form.action, formData, function () {
+            if (200 === this.status) {
+                location.reload();
+            } else {
+                alert('Error while processing the request');
+                location.reload();
+            }
+        });
+    });
+
+    if (document.createElement('datalist') && window.HTMLDataListElement) {
+        const userField = form.querySelector('input[name="user"]');
+        const userList = form.querySelector('datalist[id="userlist"]');
+
+        userField.addEventListener('keyup', function () {
+            if (userField.value.length < 2) {
+                return;
+            }
+
+            const body = {
+                FORM_SUBMIT: 'datalist_members',
+                REQUEST_TOKEN: form.querySelector('input[name="REQUEST_TOKEN"]').value,
+                value: userField.value
+            };
+
+            request('POST', form.action, new URLSearchParams(body), function () {
+                userList.innerHTML = '';
+                JSON.parse(this.response).forEach(item => {
+                    const option = document.createElement('option');
+                    option.value = item;
+                    userList.appendChild(option);
+                });
+            }, false);
+        });
+    }
+});
+{% endblock %}
+
+{% block style %}
+    <style>
+        .cto-toolbar__section--authenticator {
+
+        }
```

Comment:
Is this CSS rule necessary if there are no declarations?

Reply:
These changes are postponed now :)